### PR TITLE
Realease Attach artifacts to Maven Central

### PR DESCRIPTION
**Pull request by abhinayagarwal**
This PR releases artifacts to Maven Central instead of Gluon Nexus

### CODE REVIEW

File: .ci/release.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+fi
+
+# Release artifacts
+./gradlew publish closeAndReleaseRepository -i -PsonatypeUsername=$SONATYPE_USERNAME -PsonatypePassword=$SONATYPE_PASSWORD -Psigning.keyId=$GPG_KEYNAME -Psigning.password=$GPG_PASSPHRASE -Psigning.secretKeyRingFile=secring.gpg
+
+# Update version by 1
+newVersion=${TRAVIS_TAG%.*}.$((${TRAVIS_TAG##*.} + 1))
+
+# Update project version to next snapshot version
+sed -i -z "0,/version = $TAG/s//version = $newVersion-SNAPSHOT/" gradle.properties
```
Comment (tiainen) [inline .ci/release.sh]: Should `$TAG` be `$TRAVIS_TAG`?

File: .ci/release.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+git commit pom.xml -m "Prepare development of $newVersion"
```
Comment (tiainen) [inline .ci/release.sh]: Commit `gradle.properties` instead of `pom.xml`.

Comment (abhinayagarwal) [reply .ci/release.sh]: Fixed. Thanks for keeping a keen eye while reviewing :)

File: .ci/release.sh (hunk 1 above)
Comment (abhinayagarwal) [reply .ci/release.sh]: Fixed

**Review by tiainen — APPROVED** [review]

**Pull request merged**